app/src/main/java/com/simplecity/amp_library/ui/views/ThemedStatusBarView.java

```diff
@@ -40,23 +40,11 @@ protected void onAttachedToWindow() {
         // So, we'll just do a take(1), and since we're calling from the main thread, we don't need to worry
         // about distinctToMainThread() for this call. This prevents the 'flickering' of colors.
 
-        Aesthetic.get(getContext())
-                .colorStatusBar()
-                .take(1)
-                .subscribe(
-                        ViewBackgroundAction.create(this), onErrorLogAndRethrow()
-                );
-
-        bgSubscription = Aesthetic.get(getContext()).colorStatusBar()
-                .compose(Rx.distinctToMainThread())
-                .subscribe(
-                        ViewBackgroundAction.create(this), onErrorLogAndRethrow()
-                );
     }
 
     @Override
     protected void onDetachedFromWindow() {
-        bgSubscription.dispose();
+        //bgSubscription.dispose();
```

Comment:
Please remove this entire method. Can we remove declaration of `bgSubscription` too?